scripts/nebula.service

```diff
@@ -216,6 +216,11 @@ function status_daemon {
 
     [[ -f ${config} ]] || ERROR_AND_EXIT "${config} not found"
 
+    if [[ ${VERBOSE} -ne 0 ]]; then
+        local daemon_version=$(daemon_version ${INSTALL_ROOT}/bin/${daemon_name})
+        daemon_name="${daemon_name} version ${daemon_version}"
```

Comment:
Maybe, the following format of version info is better, and this can be always printed rather than verbose mode:
```
[INFO] nebula-storaged(977a9cbc): Running as 11068, Listening on 9779
```

Reply:
ok, comments addressed.
```
jtcheng@wattles1:/usr/local/nebula/scripts$ sudo ./nebula.service status all
[INFO] nebula-metad(977a9cbc): Running as 10926, Listening on 9559
[INFO] nebula-graphd(977a9cbc): Running as 11017, Listening on 9669
[INFO] nebula-storaged(977a9cbc): Running as 11068, Listening on 9779
```